### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by anuragcs** [Rust-GCC/gccrs#1987] [General]
Hello everyone,
I am a second year college student proficient in c++ language and also have sufficient skills in HTML,CSS,JAVASCRIPT. 
I am new to open source and want to become a good contributor. It will be great help if anyone can support and point out to me some basic issues which can get me started.

**Comment by CohenArthur** [comment]
Hi @anuragcs :D Glad to have you here! You can browse our issues which have the `good-first-pr` label: https://github.com/rust-GCC/gccrs/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-pr. You can pick whichever you like most and get started on it.

The first important step for contributing will be cloning the repo and then following our instructions for building the project. This is already a technical challenge in itself :D

Feel free to come onto our Zulip channel (link in the README.md file) and chat to us to get some help or talk about the project :D
